### DISCUSSION TITLE
Create a Discord server and other social media accounts

### DISCUSSION THREAD

**Discussion by Panquesito7** [devicons/devicon#537] [Ideas]
It would be nice if we could have a Discord channel for other users to discuss and talk (off-topic talks, icons, etc.).
We can also create a Twitter and a [Subreddit](https://reddit.com) account. I'd like to maintain the Discord server, and if possible, the Twitter and the Subreddit account. 🙂

What do you think about this? @amacado @Thomas-Boi.

**Comment by Thomas-Boi** [comment]
I'm on board with the Discord server 😃. I think it's a good thing to keep in touch with each others for details that isn't suitable on here.

I'm also fine with Twitter and the subreddit. However, I think it might be a bit too much to have both since I don't think we need to announce a lot of things. Perhaps Twitter is enough? Still, if you want to maintain them I'm cool with that 👍.

**Comment by amacado** [comment]
Personally I don't see a need for a Discord, Subreddit or Twitter account. I have multiple scenarios in mind: 

1) It's not used at all because there is no need
2) It's intransparent and hard to maintain (which information is where? Github issues? Subreddit? Discord?). It would require rules/guidelines and a structured, easy way for newbies to get into it (even today there are often pull requests/issues where people did not read the guidelines and introducing additional channels could make this even worse)
3) Specially for twitter: Who is responsible for the posts/opinions beeing posted? Since this is a community project there will always be multiple opinions to a topic and I think for a twitter account it's hard to show a differentiation

I will not block or prevent the idea and appreciate your effort. Maybe I'm just a little old fashioned ;-) Those are just my personal thoughts about the idea. 

Maybe we could think about "unofficial" devicon accounts? So the naming of discord/twitter/subreddit/.. would clearly state that everything posted is not a "official" opinion? You could compare this approach to a fan page.. (we still can link this from our repo)

**Reply by Panquesito7** [reply]
Probably only Twitter and Discord are OK. I think I'll make an unofficial Discord server for now (if you're OK with this). I'll invite you as well. About the Twitter/Subreddit, that probably can be done in the future. @Thomas-Boi I'll invite you too. What's your Discord username (you should probably send me an e-mail about this)? 🙂
